packages/salesforcedx-vscode/CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# 51.9.0 - Month DD, YYYY
+
+## Fixed
+
+#### salesforcedx-vscode-apex
```

Comment:
Pending addition of https://github.com/forcedotcom/salesforcedx-vscode/pull/3147/files - which we're hoping to get in today's release as well.